sklearn/metrics/tests/test_ranking.py

```diff
@@ -1126,8 +1126,9 @@ def test_average_precision_score_pos_label_errors():
     y_true = np.array([[1, 0], [0, 1], [0, 1], [1, 0]])
```

Comment:
could we make a separate case where we have mutliclass `y_true`, e.g. `y_true = [0, 1, 2, 0, 1, 2]`